packages/esm-tb-app/src/views/dashboard/summary-tiles/tb-summary-tiles.component.tsx

```diff
@@ -1,13 +1,26 @@
 import { OHRIProgrammeSummaryTiles } from '@ohri/openmrs-esm-ohri-commons-lib';
-import React, { useEffect, useMemo, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { getActiveDsCasesCount, getActiveDrCasesCount, } from '../../../api/api';
 
 function TbSummaryTiles({ launchWorkSpace }) {
   const { t } = useTranslation();
-  const [activeDSClientsCount, setActiveDSClientsCount] = useState(35);
-  const [activeDRClientsCount, setActiveDRClientsCount] = useState(5);
-  const tiles = useMemo(
```

Comment:
Why are you removing the `useMemo` hook?

Reply:
The `useMemo` hook was not returning values as expected. However, I have included `activeDSClientsCount` and `activeDRClientsCount` in the dependency array of the hook to fix that issue.